website/quickfiles/views.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+import requests
+from flask import send_from_directory, Response, stream_with_context
+
+from website.settings import EXTERNAL_EMBER_APPS, PROXY_EMBER_APPS
+
+quickfiles_dir = os.path.abspath(os.path.join(os.getcwd(), EXTERNAL_EMBER_APPS['quickfiles']['path']))
```

Comment:
Minor: Better to use `settings.APP_PATH` here.

Certainly not a blocker, though.